painters/train_cnn.py

```diff
@@ -24,92 +27,95 @@
 PENULTIMATE_LAYER = 51
 PENULTIMATE_SIZE = 2048
 SOFTMAX_LAYER = 55
-SOFTMAX_SIZE = 1584
+SOFTMAX_SIZE = 1057
 
 
 def _train_model():
     data_info = load_organized_data_info(IMGS_DIM_3D[1])
+    print("data_info loaded")
```

Comment:
Prefer using the `logging` library to print statements in production code: https://docs.python.org/3/library/logging.html. `logging.info()` should work here.